decider-bcl2fastq/src/main/java/ca/on/oicr/pde/deciders/Bcl2fastqDecider.java

```diff
@@ -92,6 +93,7 @@
     private Boolean isDemultiplexSingleSampleMode = false;
     private Boolean noLaneSplittingMode = false;
     private Boolean ignoreLaneSkip = false;
+    private Boolean doNotProvisionOutUndetermined = false;
```

Comment:
These variable names (`noLaneSplittingMode`, `ignoreLaneSkip`, and `doNotProvisionOutUndetermined`) are all double-negatives. It might be a big pedantic but can you switch them around so that the defaults are true?